### PR TITLE
Add devise

### PR DESCRIPTION
**Pull request by lmunozm1702**
### Can register a new user.
image

image

### User logs in with a combination of email and password.

image

image

### Hashed password should be stored in the database.

image

image

image

> Correct email:
image

image

### Can reset password.
image

image

image

image

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,17 @@
 class User < ApplicationRecord
+  attr_accessor :email_confirmation
+
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (AshakaE) [inline app/models/user.rb]: - [x] Kindly enable `:confirmable` for devise

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @lmunozm1702  & @tamana-Balkhi  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- So far so good, proper Git flow used
- I appreciate the screenshots 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (lmunozm1702) [reply app/models/user.rb]: Done!.
- Email sent to confirmate the new account
image

- New account confirmed:
image

- Activate and style for "Resend confirmation instructions"
image

image

image

**Review by fobadara — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**